components/charts/AdvancedTimeline.tsx

```diff
@@ -365,20 +410,24 @@ export default class AdvancedTimeline extends React.PureComponent<Props, State>
     const leakStart = Math.max(xScale(leakPeriodDate), xRange[0]);
 
     const leakWidth = xRange[xRange.length - 1] - leakStart;
-    if (leakWidth < 0) {
+    if (leakWidth < 1) {
       return null;
     }
+
     return (
       <ThemeConsumer>
         {(theme) => (
-          <rect
-            className="leak-chart-rect"
-            fill={theme.colors.leakPrimaryColor}
-            height={yRange[0] - yRange[yRange.length - 1]}
-            width={leakWidth}
-            x={leakStart}
-            y={yRange[yRange.length - 1]}
-          />
+          <>
```

Comment:
Should it be a svg group: `<g>` ?

Reply:
It's not necessary for the structure of the svg, so might as well not add one :shrug: